impl/src/main/java/org/jboss/fuse/wsdl2rest/impl/ResourceMapperImpl.java

```diff
@@ -29,7 +29,7 @@
     private Pattern httpPostWordsPattern = Pattern.compile(httpPostWords);
     private Pattern httpDeleteWordsPattern = Pattern.compile(httpDelWords);
     private Pattern httpPutWordsPattern = Pattern.compile(httpPutWords);
-    private Pattern resourcePattern = Pattern.compile("[a-z]+|([A-Z][a-z]+)*");
+    private Pattern resourcePattern = Pattern.compile("[A-Za-z_]+|([A-Z_][a-z_]+)*");
```

Comment:
I tried to hack this resource pattern as it seems that _ should be allowed (as per the wsdl provided for the reported issue)
where is the corresponding specification for this resource stuff? Can we have also other characters?
I don't understand why modifying this is breaking the tests